### ISSUE TITLE
i3 workaround for color picker window not marked as "tool window popup"

### ISSUE THREAD

**Issue by mo** [anseki/vscode-color#50]
When you're using a tiling window manager like i3 on linux the color picker comes up as a regular window which means it takes up half the screen space. The color picker in atom doesn't do that.

This issue can be worked around in the i3 config file, by adding this:
for_window [class="vscode-color-ui"] floating enable

![image](https://user-images.githubusercontent.com/7117/28647396-3e3c8e70-7267-11e7-8b9b-59f22e895562.png)

**Comment by anseki** [comment]
Hi @mo, thank you for the report.
So, this issue is solved by your config in the i3?

**Comment by mo** [comment]
I had to set:

  "colorHelper.disableShadow": true

in settings.json as well but now it works really well for me. It would have been better if it "just worked" out of the box but I'm happy now at least. I opened this issue mostly so that people that run into the same problem could find something useful on google to help them out. Feel free to close this issue if you feel that it's to niche to be worth fixing.
